pkg/engine/update.go

```diff
@@ -229,10 +245,30 @@ func installAndLoadPolicyPlugins(plugctx *plugin.Context, policies []RequiredPol
 			return err
 		}
 
-		_, err = plugctx.Host.PolicyAnalyzer(tokens.QName(policy.Name()), policyPath, opts)
+		analyzer, err := plugctx.Host.PolicyAnalyzer(tokens.QName(policy.Name()), policyPath, opts)
 		if err != nil {
 			return err
 		}
+
+		configFromAPI, err := plugin.ParsePolicyPackConfigFromAPI(policy.Config())
+		if err != nil {
+			return err
+		}
+
+		// If we have configuration from the API, pass it to the analyzer.
+		if len(configFromAPI) > 0 {
+			analyzerInfo, err := analyzer.GetAnalyzerInfo()
+			if err != nil {
+				return err
+			}
+			config, err := plugin.ReconcilePolicyPackConfig(analyzerInfo.Policies, configFromAPI)
+			if err != nil {
+				return errors.Wrapf(err, "reconciling configuration for analyzer %q", analyzerInfo.Name)
+			}
```

Comment:
Not: An alternative to retrieving the metadata and reconciling the config here would be to just pass the config directly from the service (or from a `--policy-pack-config` specified file) to the Policy Pack. However, then each language SDK for policies would have to handle reconciling its default values with the specified config. Looking ahead to implementing the SDK for Python, .NET, and Go, it seemed "better" to have a single implementation here for doing this reconciliation, rather than writing it N times for each language.